tests/optimizer/test-function-eliminator-double-parsed-correctly-output.js

```diff
@@ -0,0 +1,14 @@
+// EMSCRIPTEN_START_ASM
+var asm = (function(global, env, buffer) {
+ "use asm";
+ var e = 0;
+ 
+// EMSCRIPTEN_START_FUNCS
+function a() {
+ var c = 0.0;
```

Comment:
Oh wait, I see that the test runner calls the optimizer with last. So this looks ok.